docs/src/main/paradox/kubernetes-lease.md

```diff
@@ -0,0 +1,171 @@
+# Kubernetes Lease
+
+@@@ warning
+
+This module is marked as @extref:[May Change](akka:common/may-change.html)
+The API, configuration and behavior may change based on feedback from initial usage.
+
+@@@
+
+This module is an implementation of an [Akka Coordination Lease](https://doc.akka.io/docs/akka/current/coordination.html#lease) backed 
+by a [Custom Resource Definition (CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) in Kubernetes.
+Resources in Kubernetes offer [concurrency control and consistency](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) 
+that have been used to build a distributed lease/lock.
+
+A lease can be used for:
+
+* @extref[Split Brain Resolver](akka:split-brin-resolver.html). An additional safety measure so that only one SBR instance can make the decision to remain up.
+* @extref:[Cluster Singleton](akka:cluster-singleton.html#lease). A singleton manager can be configured to acquire a lease before creating the singleton.
```

Comment:
do these links end up in the right place? `typed/cluster-singleton.html` ?

Reply:
Ah they go to 2.5! Will fix that